searchResults.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>UrSpace | Home</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
+    <link rel="stylesheet" href="./dist/css/styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <script src="https://kit.fontawesome.com/c609dd8d17.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <header class="bg-primary">
+        <div class="container d-flex justify-content-between align-items-center p-3">
+            <div>
+                <a href="index.html">
+                    <img src="images/UrSpace Logo w.svg" alt="UrSpace Logo">
+                </a>
+            </div>
+            <div class="user-info"></div>
+        </div>
+    </header>
+
+    <main class="container">
+        <div class="row page-columns gap-3">
+
+            <div class="col py-4">
+                <nav class="nav position-sticky"></nav>
+            </div>
+
+            <div class="col-lg-8">
+
+                <form class="form-search mt-4 sticky-top">
+                    <label for="search" class="form-label mb-0 hide-label" aria-label="search">
+                        Search:
+                    </label>
+                    <div class="d-flex">
+                        <input type="search" class="form-control shadow-none border-primary rounded-0 rounded-start"
+                            id="searchInput" placeholder="Search for post title, ID, tags or author..." minlength="3">
+                        <button class="btn btn-primary bg-primary border-primary text-white rounded-0 rounded-end"
+                            id="searchButton"><i class="fa-solid fa-magnifying-glass"></i></button>
+                    </div>
+                </form>
+
+                <h1 class="my-3 h3 scroll-target border-bottom pb-2">Search results</h1>
+
+                <div class="message-container message-search"></div>
+
+                <div id="searchResults" class="search-results posts-container">
+                    <p>Search results appear here</p>
+                </div>
+
+                <div class="d-flex justify-content-end">
+                    <button class="scroll-button btn btn-outline-secondary rounded-5" style="display: none;"><i
+                            class="fa-solid fa-arrow-up"></i></button>
+                </div>
+
+                <button id="loadMoreButton" class="btn btn-outline-primary m-auto mb-5 w-50 justify-content-center"
+                    style="display: none;">More
+                    results...</button>
+
+            </div>
+
+            <div class="col"></div>
+
+        </div>
```

Comment:
Indentation - Please run a formatter on this file again

Reply:
The <main> is divided into three columns, so line 67 is the 3rd column but it has no content on the this page and profile page. Removing it causes the 2nd column to not be centered.
I suppose there are better ways to do this but I haven't looked for it yet.